src/main/java/hadler/ResourceHandler.java

```diff
@@ -0,0 +1,28 @@
+package hadler;
+
+import utils.FileIoUtils;
+import http.HttpRequest;
+import http.HttpResponse;
+import http.MIME;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public class ResourceHandler implements Handler{
+    public HttpResponse handle(HttpRequest httpRequest) {
+        String path = "./static";
+        MIME mime = httpRequest.getUri().getExtension().get();
+        if (mime.isTemplate()) {
+            path = "./templates";
+        }
+
+        byte[] body = new byte[0];
+        try {
+            body = FileIoUtils.loadFileFromClasspath(path + httpRequest.getUri().getPath());
+        } catch (IOException | URISyntaxException e) {
+            System.out.println(e);
+        }
+
+        return HttpResponse.staticResource(body, mime);
```

Comment:
아래와 같이 변경해보면 조금 더 깔끔해질 것 같아요. 🙂

```suggestion
        try {
            byte[] body = FileIoUtils.loadFileFromClasspath(path + httpRequest.getUri().getPath());
            return HttpResponse.staticResource(body, mime);
        } catch (IOException | URISyntaxException e) {
            System.out.println(e.getMessage());
            throw NotFoundException(); // 해당 예외를 HttpStatusCode가 404인 응답을 HttpResponse를 반환
        }

        return HttpResponse.staticResource(body, mime);
```